CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# rubocop-github
+
+## Unreleased
+
+- Unset `DisabledByDefault: true` in `config/default.yml`. This will use RuboCop's choice of default cops to enable, plus our existing augmentations in `config/default.yml`. Prevents confusing behaviour where downstream consumers didn't realise that RuboCop's default cops weren't being applied. (https://github.com/github/rubocop-github/pull/119)
```

Comment:
Do you want to be more explicit about this in the changelog?

> we've mimicked `DisabledByDefault` by explicitly marking `Enabled: false` for all the cops that weren't previously in the `config/default.yml` configuration file. **This has the effect that downstream consumers of this gem won't be surprised by new linting violations when they bump the version of this gem in their projects.**

It would good if the changelog contained similar "don't panic - no behaviour change" messaging.

Reply:
Yep, I'll write more words here.